graph/patterns/operations.md

```diff
@@ -33,32 +34,34 @@ You can consider related patterns such as [long running operations](./long-runni
 
 - Microsoft Graph supports the use of optional parameters. The optional parameter annotation can be used instead of creating function or action overloads when unnecessary.
 
-- API designer **MUST** use POST to call operations on resources.
+- API designer **MUST** use POST to call actions on resources.
+- API designer **MUST** use GET to call functions on resources.
 
 - The addition of a new mandatory not-nullable parameter to an existing action or function is a breaking change and is not allowed without proper versioning.
```

Comment:
```suggestion
- The addition of a new mandatory not-nullable parameter to an existing action or function is a breaking change and is not allowed without proper versioning that is in accordance with our [deprecation guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/graph/deprecation.md).
```

If there's something about versioning on parameters, I think we should add it to this document, but I think it's just our standard deprecation process